test/onnx/onnx_test_common.py

```diff
@@ -82,6 +82,39 @@ def run_model_test(test_suite: _TestONNXRuntime, *args, **kwargs):
     return verification.verify(*args, options=options, **kwargs)
 
 
+def assert_dynamic_shapes(
```

Comment:
IIUC, this assertion also takes care of static shape test case, which I think it's great, as for dynamic shapes we have `additional_test_input` to guard, but the for static shape, we didn't have any test to make sure it doesn't change to dynamic at the very begining by dynamo. 

Could you include `static_shape` in the name? Maybe `assert_dynamic_or_static_shapes`? or something similar.

Reply:
Indeed, it validates `ExportOptions.dynamic_shapes` bool flag, regardless of its `True` or `False` value